.drone/config.yaml

```
@@ -0,0 +1,48 @@
+---
+kind: pipeline
+type: docker
+name: default
+
+trigger:
+  ref:
+  - refs/heads/test/**
+  - refs/heads/master
+
+workspace:
+  base: /home/runner/builds
+  path: kubecf
+
+clone:
+  disable: true
+
+# Dockerfile source - https://github.com/f0rmiga/bazel-docker-image.
+.bazel_image: &bazel_image thulioassis/bazel-docker-image:1.2.1
```

Would it be better if we have a section of junk like this?

```suggestion
.anchors:
  bazel_image: &bazel_image thulioassis/bazel-docker-image:1.2.1
```